app/src/app.tsx

```diff
@@ -99,7 +100,7 @@ export const App = (): React.ReactElement => {
     setRandomTokenTransfer(null);
     setMostTradedTokenTransfers(null);
     setSponsoredToken(null);
-    setUrlString('date', getUrlDateString());
+    setUrlString('date', startDateString);
```

Comment:
this dunction should not be called anymore it should update automatically when you use `setStartDateString`